app/views/surveys/_form.html.erb

```diff
@@ -2,41 +2,35 @@
   <%= f.hidden_field :id, name: "survey[id]" %>
   <div class="row">
     <div class="col">
-      <div class="form-group">
-        <label for="description">Titel (deutsch)</label>
-        <%= f.text_field :title_de, name: "survey[title][de]", class: "form-control" %>
-      </div>
-      <div class="form-group">
-        <label for="description">Titel (polnisch)</label>
-        <%= f.text_field :title_pl, name: "survey[title][pl]", class: "form-control" %>
-      </div>
+      <% @languages.each do |language| %>
+        <div class="form-group">
+          <label for="description">Titel (<%= language %>)</label>
+          <%= f.text_field "title_#{language}", name: "survey[title][#{language}]", class: "form-control" %>
```

Comment:
es würden ab sofort daten mit keys "deutsch" und "polnisch" im system landen und nicht mehr

<img width="459" alt="Bildschirmfoto 2021-07-20 um 10 48 31" src="https://user-images.githubusercontent.com/1942953/126291272-c2288cba-3546-4ac6-95e1-a3deeb7cf797.png">

das hat bestimmt auch seitens app folgen, oder @digorath? die change kann also so nicht einfach geschehen.
wir brauchen hier die language codes und nicht die ausgeschriebene sprache.
was vom server kommt, muss also sogesehen einen sprache liefern wie "deutsch" und einen sprachencode wie "de", würd ich denken.

Reply:
derzeit ist im app code: 
```ts
export const useSurveyLanguages = () => ['de', 'pl'];
```
was z.B. so benutzt wird:
```jsx
const languages = useSurveyLanguages();
(....)
<RegularText>{responseOption.title[languages[0]]}</RegularText>
```

Derzeit werden keine von außen bestimmten sprachen genutzt, da die struktur unklar war zu dem zeitpunkt, zu welchem es in der app implementiert wurde.

Reply:
somit darf es nicht zu "deutsch" und "polnisch" werden, da sonst app update benötigt würde.

Reply:
sehe ich auch so, ja

Reply:
merk grad dass auf int/development folgendes hinterlegt ist:

![Bildschirmfoto 2021-07-20 um 11 08 02](https://user-images.githubusercontent.com/1942953/126294321-37e6e0ff-9275-47ea-abff-e1519ad8c6ef.png)

das würde für die formulare passen, nicht jedoch in ausgabestrings.
wir brauchen hier ein array von objekten schlage ich vor. das werde ich aber jetzt nicht implmentieren und den PR somit offen lassen, bis wir das richtig angehen können mit allen eventualitäten.

Reply:
ja, lass uns optimierungen/implementierungen für flexiblere sprachen hinten anstellen.
wichtig ist, dass es mit de und pl erstmal funktioniert.

alles weitere muss separat aufgearbeitet werden